zenodo/modules/profiles/templates/zenodo_profiles/profile.html

```diff
@@ -0,0 +1,150 @@
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+-#}
+
+{% extends "zenodo_theme/page.html" %}
+
+{%- block page_body %}
+<div class="container">
+  <div class="row" id="researcher-profile">
+    <div class="col-sm-3 col-xs-12" id="profile" style="margin-bottom:20px">
+      <br />
+      {%- if profile_pic_url %}
+      <img height="400px" width="400px"
+        alt="Profile Picture" class="img-responsive img-rounded center-block"
+        src="{{ profile_pic_url }}"
+        alt="Profile Picture" />
+      {%- endif %}
+
+      <div>
+        <h1 style="line-height:0.8">
+          <strong>
+            {%- if user.profile.full_name %}
+            {{ user.profile.full_name }}
+            {%- else %}
+            {{ user.profile.username }}
+            {%- endif %}
+          </strong>
+          {%- if user.profile.full_name %}
+          <small><br />{{ user.profile.username }}</small>
+          {%- endif %}
+          {%- if user.researcher_profile.affiliation %}
+          <small><br />{{ user.researcher_profile.affiliation }}</small>
+          {%- endif %}
+        </h1>
+      </div>
+      {%- if user.remote_accounts %}
+      {%- for account in user.remote_accounts %}
+      <br />
+      <div>
+          {%- if account.client_id == config.ORCID_APP_CREDENTIALS.consumer_key %}
+          <div style="margin-bottom: 4px">
+            <img width="26px" style="vertical-align:middle; margin-left: 5px" src="{{ url_for('static', filename='img/orcid.png') }}" alt="Orcid" />
+            <a class="no_hover" href="https://orcid.org/{{ account.extra_data.get('orcid') }}">
+              <span style="margin-left: 3px;font-size: 16px">{{ account.extra_data.get('orcid') }}</span>
+            </a>
+          </div>
+          {%- endif %}
+
+          {%- if account.client_id == config.GITHUB_APP_CREDENTIALS.consumer_key %}
+          <div>
+            <i style="vertical-align:middle" class="fa fa-github fa-2x fa-fw" aria-hidden="true"></i>
+            <a class="no_hover" href="https://github.com/{{ account.extra_data.get('login') }}">
+              <span style="font-size: 16px">{{ account.extra_data.get('login') }}</span>
+            </a>
+          </div>
+          {%- endif %}
+      </div>
+      {%- endfor %}
+      {%- endif %}
+
+      {%- if current_user.id != user.id %}
+        {%- if user.researcher_profile.allow_contact_owner %}
+        <div style="margin-top:18px">
+          <a class="no_hover" href="/profile/{{ user.id }}/contact">
+            <button type="button" class="btn btn-default btn-lg btn-block">Contact</button>
+          </a>
+        </div>
+        {%- endif %}
+      {%- else %}
+      <div style="margin-top:18px">
+        <a class="no_hover" href="/account/settings/profile">
+          <button type="button" class="btn btn-default">Edit Profile</button>
```

Comment:
This button can probably be removed, since the user can already click on the top right of the page on his email address to go to his Profile's settings.